bot/exts/filters/webhook_remover.py

```diff
@@ -72,7 +75,10 @@ async def on_message(self, msg: Message) -> None:
 
         matches = WEBHOOK_URL_RE.search(msg.content)
         if matches:
-            await self.delete_and_respond(msg, matches[1] + "xxx")
+            async with self.bot.http_session.delete(msg.content) as resp:
```

Comment:
Won't `msg.content` be the full message here? I think you should be using `matches[1]`, as that's the actual webhook url.

Reply:
Ah, it seems the regex doesn't match the webhook secret, just the url and ID. This will need to be changed

Reply:
Yes, how foolish of me. But I believe the full url is `matches[0]`